### PR TITLE
[PLAY-1355] Upgrade phone number lib

### PR DESCRIPTION
**Pull request by markdoeswork**
**What does this PR do?** A clear and concise description with your runway ticket url.

Runway: https://nitro.powerhrg.com/runway/backlog_items/PLAY-1355

The story was to try and update to the latest `23` and our current version is `17`

I ran into lots of breaking issues that i just upgraded to `18`. There was a decent amount of css I needed to change because the html structor updated.

1. Go to [/kits/phone_number_input/react](https://pr3453.playbook.beta.px.powerapp.cloud/kits/phone_number_input/react)
2. The kit should not change

#### Checklist:
- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [😈] **TESTS** I have added test coverage to my code.

### CODE REVIEW

File: playbook/app/pb_kits/playbook/pb_phone_number_input/_phone_number_input.scss [hunk 1]
```diff
@@ -55,7 +59,7 @@ $flag-min-resolution: 192dpi;
   }
 
   .iti__flag {
-    background-image: url("https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.19/img/flags.png");
+    background-image: url("https://intl-tel-input.com/intl-tel-input/img/flags.png");
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_phone_number_input/_phone_number_input.scss:62]: Screenshot 2024-06-06 at 2 11 03 PM

Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/pb_phone_number_input/_phone_number_input.scss:62]: Or most of them do. There's a few that are correct: Canada, Brazil... but the majority I found are incorrect @markdoeswork @co1in-1ewis

Comment (markdoeswork) [reply playbook/app/pb_kits/playbook/pb_phone_number_input/_phone_number_input.scss]: im using the correct images now https://cdnjs.com/libraries/intl-tel-input

**Comment by nidaqg** [comment]
Looks like the flags are still not correct @markdoeswork : 

Screenshot 2024-06-10 at 10 43 24 AM

File: playbook/webpack.config.js [hunk 2]
```diff
@@ -1,6 +1,4 @@
 const path = require('path')
-
-const webpack = require('webpack')
```
Comment (thestephenmarshall) [inline playbook/webpack.config.js:3]: Why was this removed?

File: playbook/webpack.config.js [hunk 3]
```diff
@@ -127,10 +123,6 @@ const SVG_URL_LOADER = {
   ],
 }
 
-new webpack.DefinePlugin({
-  'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
-})
-
```
Comment (thestephenmarshall) [inline playbook/webpack.config.js:133]: Was this removed out of necessity?

**Review by thestephenmarshall — CHANGES_REQUESTED** [review]
@markdoeswork Please only make necessary changes to this file. Do not remove comments the comments. Only affect the areas you need to (additional loader and spread at the output.

**Comment by markdoeswork** [comment]
> Looks like the flags are still not correct @markdoeswork :
> 
> Screenshot 2024-06-10 at 10 43 24 AM

They work locally but don't work in the review env. Trying to figure it out now

File: playbook/webpack.config.js (hunk 3 above)
Comment (markdoeswork) [reply playbook/webpack.config.js:133]: no its back now

**Review by thestephenmarshall — COMMENTED** [review]
Thanks for reverting the non-contextual changes in `playbook/webpack.config.js` @markdoeswork

**Review by thestephenmarshall — APPROVED** [review]
Flags look to be correct this time. 🚢
